notebooks/json_blocks/WET/BLOCK-T419.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            "description": "z offset for the triplets.",
</pre><p>Mention units in description.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>